lib/knapsack_pro/config/env.rb

```diff
@@ -13,30 +13,21 @@ class Env
 
       class << self
         def ci_node_total
-          (ENV['KNAPSACK_PRO_CI_NODE_TOTAL'] ||
-            ci_env_for(:node_total) ||
-            1).to_i
+          (knapsack_or_ci_env('KNAPSACK_PRO_CI_NODE_TOTAL', :node_total) || 1).to_i
```

Comment:
What do you think of this other name?

```suggestion
          (env_for('KNAPSACK_PRO_CI_NODE_TOTAL', :node_total) || 1).to_i
```